R/createReport.R

```diff
@@ -552,11 +553,14 @@ createReport = function(txt_folder = NULL, mztab_file = NULL, yaml_obj = list(),
     ##
     ## missed cleavages per Raw file
     ##
-    if (!is.null(df_evd)) {
+    # df_evd can be NULL; that's no problem
       lst_qcMetrics[["qcMetric_MSMS_MissedCleavages"]]$setData(df_msms, df_evd)
-    } else {
-      lst_qcMetrics[["qcMetric_MSMS_MissedCleavages"]]$setData(df_msms)
-    }
+   
+    # In case missed.cleavages is not in msms but in evd 
+    # metric checks if it was already done  
+      lst_qcMetrics[["qcMetric_MSMS_MissedCleavages"]]$setData(df_evd)
```

Comment:
line 519 (`if (!DEBUG_PTXQC) df_evd = df_evd[, c("id", "contaminant")]`) above might need a change?!

Reply:
maybe that is done other places as well. Could you check?